stdlib/public/TensorFlow/TensorHandle.swift

```diff
@@ -148,16 +165,12 @@ internal extension ShapedArray where Scalar : AccelerableByTensorFlow {
     // host.
     // NOTE: This will not perform a copy if the handle is already on the host.
     let context = _ExecutionContext.global
-    let hostHandle: CTensorHandle = context.withMutableCContext { ctx in
-      debugLog("Calling TFE_TensorHandleCopyToDevice().")
-      let ret = TFE_TensorHandleCopyToDevice(
-        cTensorHandle, ctx, context.cpuDeviceName, status
-      )
-      checkOk(status)
-      internalConsistencyCheck(ret != nil,
-                               "TFE_TensorHandleCopyToDevice() returned nil.")
-      return ret!
-    }
+    debugLog("Calling TFE_TensorHandleCopyToDevice().")
+    let hostHandle: CTensorHandle? = TFE_TensorHandleCopyToDevice(
```

Comment:
The type signature here seems redundant. But if you prefer to keep it for clarity that's reasonable too!

Reply:
Yes i feel it adds clarity.